clipboard/ios/PluginTests/PluginTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+import Capacitor
+@testable import Plugin
+
+class PluginTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testEcho() {
+        // This is an example of a functional test case for a plugin.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+
+        let value = "Hello, World!"
+        let plugin = MyPlugin()
+
+        let call = CAPPluginCall(callbackId: "test", options: [
+            "value": value
+        ], success: { (result, _) in
+            let resultValue = result!.data["value"] as? String
+            XCTAssertEqual(value, resultValue)
+        }, error: { (_) in
+            XCTFail("Error shouldn't have been called")
+        })
+
+        plugin.echo(call!)
+    }
```

Comment:
```suggestion
```

remove echo test

Reply:
Do we also want to leave in the android tests? I noticed we were doing that. 

https://github.com/ionic-team/capacitor-plugins/blob/6356931b776b1e8cd0a509f79cdf37841d1037a0/clipboard/android/src/androidTest/java/com/getcapacitor/android/ExampleInstrumentedTest.java

https://github.com/ionic-team/capacitor-plugins/blob/6356931b776b1e8cd0a509f79cdf37841d1037a0/clipboard/android/src/test/java/com/getcapacitor/ExampleUnitTest.java

Reply:
at the moment we are leaving them, but they are pointless, so we could remove them
but at least they pass, on iOS they fail because the echo method was deleted, and also the PluginTests.swift has changed a lot since you created this and it's out of date with the template